packages/datepicker-react/src/DatePicker.tsx

```diff
@@ -112,31 +113,45 @@ export function DatePicker({
 
     return (
         <div className={`jkl-datepicker ${className}`} ref={datepickerRef}>
-            <TextField
-                placeholder={placeholder}
-                label={label}
-                type="text"
-                value={dateString}
-                onChange={onInputChange}
-                onFocus={openDatepicker}
-                data-testid="jkl-datepicker-input"
-            />
-
-            <div hidden={datepickerHidden}>
-                <CoreDatepicker
-                    timestamp={date ? date.getTime() : undefined}
-                    months={months}
-                    days={days}
-                    onDatepickerClickDay={onClickCalendarDay}
-                    className="jkl-datepicker__calendar"
-                >
-                    <div className="jkl-datepicker__calendar-header">
-                        <TextField label={yearLabel} type="year" className="jkl-datepicker__calendar-header--year" />
-
-                        <Select className="jkl-datepicker__calendar-header--month" label={monthLabel} items={[]} />
-                    </div>
-                    <table data-testid="jkl-datepicker-calendar" />
-                </CoreDatepicker>
+            <div className="jkl-datepicker__outer-wrapper">
```

Comment:
outer-wrapper is a bit butter on lard right?

Reply:
Hehe, yeah.. The reason is that in `Dropdown` there is a `option-wrapper` element, and I wanted to differentiate it from that. I simply used the same naming convention here.